xml/System.Linq/ParallelEnumerable.xml

```diff
@@ -4102,6 +4103,7 @@
         <exception cref="T:System.ArgumentNullException">
           <paramref name="source" /> or <paramref name="action" /> is a null reference (Nothing in Visual Basic).</exception>
         <exception cref="T:System.AggregateException">One or more exceptions occurred during the evaluation of the query.</exception>
+        <exception cref="T:System.NotSupportedException">The exception that occurs when this method is called.</exception>
```

Comment:
Since this method always throws NotSupportedException, the other exceptions (on lines 4102 through 4105) should be removed.